jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/SelectableChannelEndPoint.java

```diff
@@ -133,7 +133,7 @@ public SocketAddress getLocalSocketAddress()
             SelectableChannel channel = getChannel();
             if (channel instanceof NetworkChannel)
                 return ((NetworkChannel)channel).getLocalAddress();
-            return super.getLocalSocketAddress();
+            return null;
```

Comment:
is returning `null` a wise thing here?
Wouldn't the HttpConnection.localAddress be appropriate here?

Reply:
The super implementation was just to return null.  I removed the default implementation to ensure it was clear that implementations needed to provide this.
So returning null is not new.